app/views/admin/words/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<% provide(:title, "Category words") %>
+
+<div class="container mt-4">
+    <div class="row justify-content-center">
+        <div class="card shadow col-md-8">
+            <div class="card-body">
+                <h1 class="mt-4"><%= @category.title %></h1>
+                <hr>
+                <table class="table table-hover">
+                    <thead>
+                        <tr>
+                            <th>Word</th>
+                            <th>Answer</th>
+                            <th>Action</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <% @words.each do |word| %>
+                        <tr>
+                            <td><%= word.name %></td>
+                            <td>answer</td>
+                            <td>
+                                <%= link_to "Edit", edit_admin_category_word_path(@category.id, word.id) %> |
+                                <%= link_to "Delete", [:admin, @category, word],
+                                method: :delete,
+                                data: { confirm: "You sure?"} %>
```

Comment:
@shaniajubelag Indent lines `25-26`, to make it clear that it is part of line `24`